Transfer/Tests/CreateTransferResponse.json

```diff
@@ -0,0 +1,47 @@
+{
+    "token": "trf-123456",
+    "status": "QUOTED",
+    "createdOn": "2016-05-01T00:00:00",
+    "clientTransferId": "6712348070812",
+    "sourceToken": "usr-ac5727ac-8fe7-42fb-b69d-977ebdd7b48b",
+    "sourceAmount": "80",
+    "sourceCurrency": "CAD",
+    "destinationToken": "trm-e3af62b4-24d5-100f-a4eb-ada857b8fc30",
+    "destinationAmount": "62.29",
+    "destinationFeeAmount": "1.20",
+    "destinationCurrency": "USD",
+    "foreignExchanges": [
+        {
+            "sourceAmount": "100.00",
+            "sourceCurrency": "CAD",
+            "destinationAmount": "63.49",
+            "destinationCurrency": "USD",
+            "rate": "0.79"
+        },
+        {
+            "sourceAmount": "120.00",
+            "sourceCurrency": "CAD",
+            "destinationAmount": "63.49",
+            "destinationCurrency": "USD",
+            "rate": "0.39"
+        },
+        {
+            "sourceAmount": "110.00",
+            "sourceCurrency": "CAD",
+            "destinationAmount": "63.49",
+            "destinationCurrency": "USD",
+            "rate": "0.88"
+        }
+    ],
+    "notes": "Partial-Balance Transfer",
+    "memo": "TransferClientId56387",
+    "expiresOn": "2016-05-01T00:02:00",
+    "links": [
+        {
+            "params": {
+                "rel": "self"
+            },
+            "href": "https://api.sandbox.hyperwallet.com/rest/v3/transfers/trf-aa308d58-75b4-432b-dec1-eb6b9e341111"
```

Comment:
localhost

Reply:
This is a temp file, will use Yurii's file later once he starts to work on unitests